src/openms/include/OpenMS/DATASTRUCTURES/FASTAContainer.h

```diff
@@ -300,6 +311,11 @@ class FASTAContainer<TFI_Vector>
     return data_.size();
   }
 
+  /// required for template parameters!
+  void reset()
+  {
```

Comment:
please insert
```
  activate_count_ = 0;
  cache_count_ = 0;
```
to reset this.
Also here, please augment the tests.
i.e. after resetting, `activateCache() == true` again

Reply:
Will do.